tests/conftest.py

```diff
@@ -201,11 +201,11 @@ def fuzzy_assert_proportion(
         if reject_null:
```

Comment:
We should release a new vivarium_validation package that includes fuzzy checker so we don't need to copy/paste this stuff everywhere. It would also allow for more streamlined updates to  validation methods. @rmudambi ?

Reply:
I agree. I've been thinking about a validation package and a data utils package. I'd like to not have to add two more packages but those two things packaged together may be weird.